services/IoT/applications/libraries/java.md

```diff
@@ -91,18 +91,18 @@ To connect to the {{site.data.keyword.iot_short_notm}} use the ``connect()`` fun
     myClient.connect();
 ```
 
-After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, your  application clients can subscribe to device events, subscribe to device statuses, and  publish device events and commands.
+After successfully connecting to the {{site.data.keyword.iot_short_notm}} service, your application clients can subscribe to device events, subscribe to device statuses, and publish device events and commands.
 
 ## Subscribing to device events
 {: #subscribing_device_events}
 
 Events are the mechanism by which devices publish data to {{site.data.keyword.iot_short_notm}}. The device controls the content of the event and assigns a name for each event that it sends.
 
-When an event is received by the {{site.data.keyword.iot_short_notm}} instance, the credentials of the received event identify the sending device, making it impossible for a device to impersonate another device.
+When an event is received by the {{site.data.keyword.iot_short_notm}} instance, the credentials of the received event identify the sending device, which makes it impossible for a device to impersonate another device.
 
 
 
-By default, applications subscribe to all events from all connected devices. Use the type, ID, event, and msgFormat parameters to control the scope of the subscription. A single client can support multiple subscriptions. The following code samples provide examples of how you can subscribe to devices that are dependent on device type, ID, event, and msgFormat parameters.
+By default, applications subscribe to all events from all connected devices. Use the type, ID, event, and msgFormat parameters to control the scope of the subscription. A single client can support multiple subscriptions. The following code samples provide examples of how you can subscribe to devices that are dependent on device type, ID, event, and msgFormat parameters. 
```

Comment:
the phrase "that are dependent on device type, ID, event, and msgFormat parameters" is kind of dangling at the end, which makes it hard to understand whether the examples are dependent on the parameters or the devices are dependent on the parameters. Needs to be clarified/rewritten.